src/main/scala/rocket/icache.scala

```diff
@@ -128,7 +128,7 @@ class ICache(latency: Int)(implicit p: Parameters) extends CoreModule()(p) with
       io.resp.bits.datablock := Mux1H(s1_tag_hit, s1_dout)
       io.resp.valid := s1_hit
     case 2 =>
-      val s2_hit = RegEnable(s1_hit, !stall)
+      val s2_hit = RegEnable(s1_hit, Bool(false), !stall)
```

Comment:
If that's the case, then the first commit probably isn't necessary (besides the change in output port and maybe the formatting change).

I need to edit this anyway, since my FIRRTL patch didn't exactly do what I thought it would.